apps/aecore/lib/aecore/chain/worker.ex

```diff
@@ -212,17 +217,21 @@ defmodule Aecore.Chain.Worker do
       end)
   end
 
-  defp get_blocks(blocks_acc, next_block_hash, size) do
+  defp get_blocks(blocks_acc, next_block_hash, final_block_hash, size) do
```

Comment:
Style: you are calling the same number (max number of blocks to return) first `count` then (inside this function) `size`. Please make the naming consistent. *Renaming from `count` to `size` looks easier.*